src/database/models/triprequest.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  const tripRequest = sequelize.define('tripRequest', {
+    leavingFrom: DataTypes.STRING,
+    goingTo: DataTypes.INTEGER,
+    travelDate: DataTypes.STRING,
+    returnDate: DataTypes.STRING,
+    travelReason: DataTypes.STRING,
+    tripType: DataTypes.STRING,
+    status: DataTypes.STRING,
+    requesterId: DataTypes.INTEGER,
```

Comment:
don't you think that tripRequests should be associated with the user, with a foreign key

Reply:
and also I think it should be associated with accommodation